Drop-In/src/main/java/com/braintreepayments/api/DropInEventType.java

```diff
@@ -0,0 +1,14 @@
+package com.braintreepayments.api;
+
+public enum DropInEventType {
+    ADD_CARD_SUBMIT,
+    CARD_DETAILS_SUBMIT,
+    DELETE_VAULTED_PAYMENT_METHOD,
+    DID_DISPLAY_SUPPORTED_PAYMENT_METHODS,
+    DISMISS_VAULT_MANAGER,
+    EDIT_CARD,
```

Comment:
```suggestion
    EDIT_CARD_NUMBER,
```
This event is fired when a user clicks into the card number edit text to edit the number right? Want to clarify it.